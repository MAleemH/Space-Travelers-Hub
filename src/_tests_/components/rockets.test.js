import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import Rockets from '../../components/rockets/rockets';
import store from '../../redux/store';

describe('Rockets', () => {
  it('Rockets Page Rendered Correctly', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Rockets />
        </Provider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
