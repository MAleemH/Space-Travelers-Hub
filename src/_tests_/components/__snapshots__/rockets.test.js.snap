// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rockets Rockets Page Rendered Correctly 1`] = `null`;
